bubble-wrap-style.yaml

```diff
@@ -3578,328 +3745,302 @@ layers:
                     mapzen_icon_library:
                         size: [[4,5px],[8,6px],[9,0px]]
                         sprite: townspot-m-rev
-                        priority: 6
                         text:
                             font:
                                 size: [[2,11px],[6,15px],[8,16px],[10,18px],[12,18px],[13,0px]]
                     text-blend-order:
-                        priority: 6
                         font:
                             size: [[2,11px],[6,15px],[8,16px],[10,18px],[12,18px],[13,0px]]
                 capital:
                     # we only want townspots to show as capital dots starting at zoom 5
-                    filter: { country_capital: true, $zoom: { min: 5 } }
+                    filter:
+                        all:
+                            - global.ux_point_of_view_kind_filter_country_capital
+                            - $zoom: { min: 5 }
                     draw:
                         mapzen_icon_library:
                             size: [[10,8px],[11,0px]]
                             sprite: capital-l
-                            priority: 5
-                        text-blend-order:
```

Comment:
Was the `text-blend-order` removed because these are handled by the icon drawing?

Reply:
It's because the master layer's default `text-blend-order` sets `priority: global.collision_priority` above, so they aren't needed every instance – since all the specific values moved server side and it's just a lookup now.